hyperspy/signal.py

```diff
@@ -4721,9 +4721,9 @@ def set_signal_type(self, signal_type=None):
 
         HyperSpy ships with a minimal set of known signal types. External
         packages can register extra signal types. To print a list of
-        registered signal types in the current installation run this method
-        without arguments. Note that
-
+        registered signal types in the current installation, call
+        :py:meth:`hyperspy.utils.print_known_signal_types`, and see
+        the developer guide for details on how to add new signal_types.
```

Comment:
I am not convinced that the developer guide is really useful here - the registration mechanism is already complicated enough for extension developer. Maybe it would be more useful to point to https://github.com/hyperspy/hyperspy-extensions-list

Reply:
The developer guide has this lengthy section, which refers to the link you mention anyway. I don't think it's an issue.

https://hyperspy.readthedocs.io/en/latest/dev_guide/writing_extensions.html

Reply:
Yes, indeed and most users will not be interested in writing extension or learning about the registration mechanism! The extension list is a user facing document while the most of the content of this "writing extension" section of the developer guide is of little relevance for most users.
On this topic, it would be worth adding a column "signal type" to https://github.com/hyperspy/hyperspy-extensions-list, so it can be updated easily and independently from hyperspy documentation.

Reply:
@ericpre ok - I've updated the docstring to include a link to the repo - is that all?